projects/wm-core/src/login/login.component.html

```diff
@@ -0,0 +1,90 @@
+<webmapp-modal-header
+  class="webmapp-login-header"
+  [title]="'modals.login.title' | wmtrans"
+  (dismiss)="dismiss()"
+>
+</webmapp-modal-header>
+<ion-content scrollY="false">
+  <form class="webmapp-login-form" (ngSubmit)="login()" [formGroup]="loginForm">
+    <h1>
+      {{ "modals.login.welcome" | wmtrans }}
```

Comment:
allora questo modo di scrivere annidata la chiave della traduzione è un lascito del vecchio programmatore che io sto eliminando, l'idea di base è usare plain object(dizionari chiave valore) dove la chiave è la traduzione in italiano, quindi in questo caso va eliminata quella chiave e va messa la traduzione in ita